src/synthcity/plugins/core/models/tabular_goggle.py

```diff
@@ -0,0 +1,194 @@
+# stdlib
+from typing import Any, Optional
+
+# third party
+import numpy as np
+import pandas as pd
+import torch
+from pydantic import validate_arguments
+
+# synthcity absolute
+from synthcity.utils.constants import DEVICE
+
+# synthcity relative
+from .goggle import Goggle, GoggleLoss
+from .tabular_encoder import TabularEncoder
+
+
+class TabularGoggle:
```

Comment:
Please add dosctrings for the parameters, some parameters are confusing. For example, `schema` seems to be the feature names.

Reply:
Added docstring. Removed schema argument, as it wasn't required.